src/Contacts/Constacts.module.css

```diff
@@ -1,3 +1,64 @@
 .title {
   color: #3f51b5;
 }
+
+.links {
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.links__item {
+  margin: 0 5px 5px 0;
+  transition: all ease .3s;
+}
+
+.links__item:hover {
+  transform: scale(.9);
+  opacity: .8;
+}
+
+.links__item:active {
+  transform: scale(1);
+  opacity: 1;
+}
+
+.avatar {
+  width: 200px;
+  height: 200px;
+  background: url('/img/avatar.png');
+  background-size: cover;
+  margin: 0 10px 0 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: nowrap;
+  margin-bottom: 20px; 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=6099781)